server/src/lib/dokku/apps/logs.ts

```diff
@@ -1,10 +1,18 @@
 import NodeSsh from 'node-ssh';
 
 const parseLogsCommand = (commandResult: string) => {
-  const allLogs = commandResult.split('\u001b');
+  const allLogs = commandResult.split('\n');
+  const logs = [];
   //We remove redundnat chars from good log & pick only 3rd item in array
   //which is a good log
```

Comment:
should we change the comment?
Can you explain the changes you did? Like this is pretty hard to understand :D